### ISSUE TITLE
Different handling of uploaded files in form plugins

### ISSUE THREAD

**Issue by PeterW-LWL** [divio/aldryn-forms#170]
In sent mails, the form plugins handle uploaded files differently:
- FormPlugin shows absolute URL to a uploaded file
- EmailNotificationForm only shows file name

The EmailNotificationForm should display the absolute URL to the uploaded file as well.

CC @FinalAngel

**Comment by wfehr** [comment]
Had the same problem some weeks ago and solved it by monkey-patching as followed:


```
from aldryn_forms.contrib.email_notifications.notification import (
    BaseNotificationConf,
)


def get_context(self, form):
    text_context = form.get_cleaned_data()
    text_context['form_name'] = self.form_plugin.name

    if self.custom_context_choices:
        custom_context = self.get_custom_context(form)
        text_context.update(custom_context)
    return text_context


BaseNotificationConf.get_context = get_context
```

**Comment by PeterW-LWL** [comment]
Thanks @wfehr this is a great workaround.

**Comment by victor-yunenko** [comment]
This problem has also been fixed the [release/6.0.x](https://github.com/divio/aldryn-forms/tree/release/6.0.x) branch 👌
